src/commands/initProjectForVSCode/pythonSteps/addPythonInitVSCodeSteps.ts

```diff
@@ -16,6 +19,7 @@ export async function addPythonInitVSCodeSteps(
     promptSteps: AzureWizardPromptStep<IProjectWizardContext>[],
     executeSteps: AzureWizardExecuteStep<IProjectWizardContext>[]): Promise<void> {
 
+    const createPythonVenv: boolean = !!getWorkspaceSetting<boolean>('createPythonVenv', context.workspacePath);
     const venvs: string[] = [];
```

Comment:
Please copy this line over from the other file, so that we can track how often people create a virtual environment
```
context.telemetry.properties.createPythonVenv = String(createPythonVenv);
```